script.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable linebreak-style */
```

Comment:
- **Your project looks great. :+1: :+1: You have done a great job, but there are few changes you need to make before moving to the next milestone.**
- [ ] As required by Microverse, you are not meant to use any linters disable rule in your code base. I'll suggest you remove this and tr to correct the issue as the linters are here to make sure your code follows the best standard practices. :+1: :+1: 
- [ ] Also it seems the `eslint` is not running on your pull request. visit the `html-css-js` [linters repository](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and follow the direction. Remember to update the content of your linters.yml file

Reply:
I don't know why I am getting this issue but the eslint works fine in my local machine either when I push my changes to the remote machine When all of the checks starts, at first the eslint linter starts working and it successfully passing, but when all the checks are finished, the eslint check suddenly disappears. 
could you please help me how to fix it?